CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+- (Breaking) The script files now end with `.ts` instead of `.js` as the implementation was converted to TypeScript
```

Comment:
Don't know if this should be considered a breaking change, since the "end-user API" here is the commands and their arguments, and they have stayed the same even though the underlying implementation has changed?

Reply:
The name of the main executable has changed. If the tool is used in a script (e.g. `./streamr.js subscribe foobar`), that script statement needs to be updated.

Reply:
Fixed